lib/ui/window.dart

```diff
@@ -393,6 +392,17 @@ class FlutterView {
   @Native<Void Function(Pointer<Void>)>(symbol: 'PlatformConfigurationNativeApi::UpdateSemantics')
   external static void _updateSemantics(_NativeSemanticsUpdate update);
 
+  /// Informs the engine whether the framework is generating a semantics tree.
+  ///
+  /// After this has been set to true, platforms are expected to prepare for accepting
+  /// semantics update sent via [updateSemantics]. When this is set to false, platforms
+  /// may dispose any resources associated with processing semantics as no further
+  /// semantics updates will be sent via [updateSemantics].
+  void setSemanticsTreeEnabled(bool enabled) => _setSemanticsTreeEnabled(enabled);
```

Comment:
nit: Should we just call this `setSemanticsEnabled`? I don't think we officially call this a tree anywhere else in the API.